### PR TITLE
expose disable metrics flag in reducer template

### PR DESCRIPTION
**Pull request by dustin-bingham**

### CODE REVIEW

File: templates/reducer-deployment.yaml [hunk 1]
```diff
@@ -36,6 +36,9 @@ spec:
             - --port={{- .Values.reducer.telemetryPort }}
             - --log-console
             - --no-log-file
+            {{- if hasKey .Values.log "level" }}
```
Comment (dustin-bingham) [inline templates/reducer-deployment.yaml:39]: Just moving this location to be near the other "log" options.

File: templates/reducer-deployment.yaml [hunk 2]
```diff
@@ -46,6 +49,9 @@ spec:
             {{- if .Values.reducer.disableInternalMetrics }}
```
Comment (dustin-bingham) [inline templates/reducer-deployment.yaml:49]: Once Samiur finishes 736, I think this flag can go away and internal metrics can be folded into the disableMetrics flag.

File: values.yaml [hunk 3]
```diff
@@ -89,6 +89,7 @@ reducer:
   matchingShards: 1
   aggregationShards: 1
   disableInternalMetrics: true
+  disableMetrics: ""
```
Comment (bjandras) [inline values.yaml:92]: Should it be a list that is `','`-joined?

Comment (dustin-bingham) [reply values.yaml:92]: Yeah, that's the intention.  Do I need to do something special up in the template, like use the quote fn? I tested without and it looks like helm did this: `'--disable-metrics=http.all, udp.drops'`

Comment (bjandras) [reply values.yaml:92]: What I meant is a yaml list, e.g.:
```
disabledMetrics:
  - http.all
  - udp.drops
```

Not sure what makes more sense.

Comment (dustin-bingham) [reply values.yaml:92]: Ohhhhhhh I see.  Well that is certainly easier to read imo - I didn't even think of that.  What would you prefer?

Comment (bjandras) [reply values.yaml:92]: The log whitelist is supposed to be a comma-separated string.

We should go with which is harder to get wrong. I think that would be a yaml list. We can even supply all possible values in `values.yaml` and comment them out.

Comment (dustin-bingham) [reply values.yaml:92]: Cool, let me work on that then.  Thanks!

Comment (bjandras) [reply values.yaml]: Yeah!

**Pull request merged**